x-pack/filebeat/module/panw/panos/test/pan_inc_threat.log-expected.json

```diff
@@ -2177,7 +2317,9 @@
             "pan-os",
             "forwarded"
         ],
-        "url.original": "-/"
+        "url.domain": "-",
+        "url.original": "-/",
```

Comment:
I'm guessing `-/` means we don't really have a URL, in that case can we skip populating `url.domain`, `related.hosts`, etc?  Maybe change if statement in grok to:

```
ctx?.url?.original != null && ctx.url.orginal != '-/'
```

Reply:
I was tempted to ignore this one, but since `/` will appear only if it is explicitly in the url and there is no special comment to omit it the initial `/`in the docs for `url.path` as there are to omit `?` or `.` for `url.query` or `url.extension` I was inclined to let it there, even though I do not have any strong opinion leaning in either direction. Maybe @ebeahan can give some advice here.

Reply:
For `url.path`, we want to retain the initial `/`. Even a single `/` character is a valid path.

Reply:
Sorry @leehinman I do not know what I was thinking when I read your comment but thought you were referring to `url.path` 🙇, it makes sense and already changed it . In any case thanks @ebeahan since you solved my own concern.